cloud-shell/cloudbuild.yaml

```diff
@@ -14,7 +14,7 @@
 
 steps:
 - name: 'gcr.io/cloud-builders/docker'
-  args: ['build', '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME', '.']
+  args: ['build', '-t', 'gcr.io/$PROJECT_ID/$REPO_NAME', 'cloud-shell/.']
```

Comment:
This will rebuild the current image from master branch. We also need to rebuild the released custom image version each time that base image changes.